src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/mk4iSwerveModule/MK4ISwerveModule.java

```diff
@@ -105,7 +106,12 @@ public void updateInputs(SwerveModuleInputsAutoLogged inputs) {
         inputs.linearCurrent = linearMotor.getSupplyCurrent();
         inputs.angularCurrent = angularMotor.getStatorCurrent();
 
-        inputs.linearMetersPassed = Conversions.MK4IConversions.convertTicksToMeters(linearMotor.getSelectedSensorPosition());
+        inputs.linearMetersPassed = SwerveUtils.getCouplingCompensatedDistance(
```

Comment:
should probably split into a line for calculating distance before coupling and then adding the coupling effect